lib/rota_alerts.js

```diff
@@ -25,8 +26,9 @@ class RotaAlerts {
     const date = moment(startDate).format("dddd Do MMMM");
     const text = this.message(date);
     const slackMessage = new SlackMessage(email, text);
+    const emailMessage = new EmailMessage(email, text);
 
-    return await slackMessage.send();
+    return await [slackMessage.send(), emailMessage.send()];
```

Comment:
I think you might need to do `await Promise.all([slackMessage.send(), emailMessage.send()])` here

Reply:
I have no idea so I'll take your word for it 😆